week-1/2-mandatory/3-function-output.js

```diff
@@ -1,24 +1,23 @@
-// Add comments to explain what this function does. You're meant to use Google!
+/******************************************
+ * This function return a positive
+ * random number less than 9 including 0
+ * ****************************************/
 function getNumber() {
   return Math.random() * 10;
 }
 
-// Add comments to explain what this function does. You're meant to use Google!
+/******************************************
+ * This function takes two strings and
+ * join (concatenate) them together
+ * ****************************************/
```

Comment:
True - but these could also be arrays :)
`[1, 2, 3].concat([6, 7, 8]) // returns [1, 2, 3, 6, 7, 8]`